src/main/java/com/example/music/backend/playlist/converter/PlaylistDtoMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.example.music.backend.playlist.converter;
+
+import com.example.music.backend.playlist.domain.Playlist;
+import com.example.music.backend.playlist.dto.PlaylistDto;
+import com.example.music.backend.user.domain.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+
+@Mapper(componentModel = "string")
+public interface PlaylistDtoMapper {
```

Comment:
Why don't use PlaylistMapper for Dto and Response?

Reply:
Now one Mapper PlaylistMapper